src/button/src/components/styled/StyledButton.js

```diff
@@ -0,0 +1,109 @@
+import styled from 'styled-components';
+import { getStyleVariation } from '../../../../utils/StyleUtils';
+
+import {
+  defaultActive,
+  defaultDisabled,
+  defaultFocus,
+  defaultHover,
+  defaultStyle,
+} from './DefaultButton';
+
+import {
+  secondaryActive,
+  secondaryDisabled,
+  secondaryFocus,
+  secondaryHover,
+  secondaryStyle,
+} from './SecondaryButton';
+
+import {
+  primaryActive,
+  primaryDisabled,
+  primaryFocus,
+  primaryHover,
+  primaryStyle,
+} from './PrimaryButton';
+
+import {
+  subtleActive,
+  subtleDisabled,
+  subtleFocus,
+  subtleHover,
+  subtleStyle,
+} from './SubtleButton';
```

Comment:
All of these `Button` files are exporting only styles, not actually a button component. I think it'd be clearer if they are renamed accordingly, i.e. something like `'./PrimaryButton' -> './PrimaryButtonStyles'`